angular.json

```diff
@@ -34,7 +34,7 @@
             "baseHref": "/"
           },
           "zh-Hans": {
-            "translation": "i18n/zh-Hans/messages.zh-Hans.xlf",
+            "translation": "i18n/zh/messages.zh.xlf",
```

Comment:
I guess it should work the other way around. `zh` is generic language code without region, so it should be mapped to a specific region. `zh-Hans` should stay as `zh-Hans`.

Reply:
I think, top level language code should have translation file, and using parent's translation or own is children's choice. Also, translation `OWNERS` should be set by who can translate realy.

Reply:
Those regional codes are there for a reason, even though we have tried to simplify it before, they can contain small differences for some words. That is why we shouldn't just take zh-XX and map it to the generic one (zh). 

It's like "mapping" car and scooter to a vehicle. You will lose features that are specific to each one of them. We should do this the other way around and say that for most people vehicle of choice (zh) will be a car (zh-Hans).